src/activities/find_the_day/ActivityInfo.qml

```diff
@@ -0,0 +1,40 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2017 Amit Sagtani <asagtani06@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import GCompris 1.0
+
+ActivityInfo {
+  name: "find_the_day/FindTheDay.qml"
+  difficulty: 4
```

Comment:
I would put 5 or 6 as it's more difficult than calendar and require more knowledge from children

Reply:
changed to 6 :+1: